index.html

```diff
@@ -21,159 +21,164 @@
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
+        <div>
+          <h2 class="heading-underline">Learn more</h2>
+
+          <div class="articlearea">
+            <div class="article">
+              <h3 class="article__title">Why do refugees need bikes?</h3>
+              <p class="article__summary">
+                Many refugees are placed in housing in areas where there are few
+                jobs. Bikes provide low-cost transportation so that they can get
+                to work, manage child care and do their shopping.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
+              </p>
             </div>
-          </div>
-        </div>
 
-        <div class="sidebar">
-          <h2 class="heading-underline">Upcoming events</h2>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
+            <div class="article">
+              <h3 class="article__title">How can I help?</h3>
               <p class="article__summary">
-                Riders needed now to help raise awareness and funds for BfR and
-                Médecins Sans Frontières.
+                We need lots of bikes and bike accessories! If you have an old
+                bike you don't use, bring it to one of our dropoff events. Or
+                come attend one of our fundraisers.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
               </p>
             </div>
           </div>
+        </div>
+      </div>
 
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
-              </h3>
-            </div>
+      <div class="sidebar">
+        <h2 class="heading-underline">Upcoming events</h2>
+
+        <div class="article sideBySide">
+          <img
+            class="article__thumbnail"
+            src="images/spring-fundraisers_thumbnail.jpg"
+            alt="pink sakura tree branch"
```

Comment:
Great catch on adding the alt text for the image! Including alt text is a great accessibility practice. Well done.